docs/docs/howto/immutable-deployments.md

```diff
@@ -0,0 +1,75 @@
+# Immutable deployments with Contrast
+
+This section provides guidance on how to setup an immutable deployment with Contrast, that is, a deployment which can't be changed after the initial setup.
+
+## Applicability
+
+In multi-party setups with (mutually) distrusting parties, ensuring that no party can alter the deployment or decrypt state can be desirable.
+
+## Prerequisites
+
+1. [Set up cluster](./cluster-setup/bare-metal.md)
+2. [Install CLI](./install-cli.md)
+3. [Deploy the Contrast runtime](./workload-deployment/runtime-deployment.md)
+4. [Add Coordinator to resources](./workload-deployment/add-coordinator.md)
+5. [Prepare deployment files](./workload-deployment/deployment-file-preparation.md)
+6. [Generate annotations and manifest](./workload-deployment/generate-annotations.md)
+
+## How-To
+
+After running the [`contrast generate`](./workload-deployment/generate-annotations.md) command, the manifest contains fields `WorkloadOwnerKeyDigest` and `SeedshareOwnerPubKeys`.
+Deleting these fields before calling [`contrast set`](./workload-deployment/set-manifest.md) will render the deployment immutable.
+
+```json
+{
+  "WorkloadOwnerKeyDigests": [
+    "<remove this digest to prevent manifest updates>" 
+  ],
+  "SeedshareOwnerPubKeys": [
+    "<remove this key to prevent state decryption and coordinator recovery>"
+  ]
+}
+```
+
+### Preventing manifest updates
+
+To prevent manifest updates, delete the `WorkloadOwnerKeyDigest`.
+Now [set the manifest](./workload-deployment/set-manifest.md) through `contrast set`.
+Once completed, the Contrast Coordinator will no longer accept the workload owner key for future manifest changes.
```

Comment:
The seedshare owner can still force manifest changes (through recovery), at least if they have access to the Coordinator's ConfigMaps.